core/tauri-utils/src/config.rs

```diff
@@ -2587,12 +2587,12 @@ impl WindowsUpdateInstallMode {
 
   /// Returns the associated nsis arguments.
   ///
-  /// [WindowsUpdateInstallMode::Passive] will return `["/P", "/R"]`
-  /// [WindowsUpdateInstallMode::Quiet] will return `["/S", "/R"]`
+  /// [WindowsUpdateInstallMode::Passive] will return `["/P", "/R", "/NS"]`
+  /// [WindowsUpdateInstallMode::Quiet] will return `["/S", "/R", "/NS"]`
   pub fn nsis_args(&self) -> &'static [&'static str] {
     match self {
-      Self::Passive => &["/P", "/R"],
-      Self::Quiet => &["/S", "/R"],
+      Self::Passive => &["/P", "/R", "/NS"],
```

Comment:
since this isn't related to the install mode, shouldn't it be set in `core/tauri/src/updater/core.rs` instead?

Reply:
that could be argued for `/R` as well, but both should stay here because it is called `WindowsUpdateInstallMode`, i.e. related to updater and these flags are in scope of the updater.

Reply:
I agree about the `/R` part but disagree about the rest. anyway I don't want to hold off on the PR any longer